client/src/pages/UserLanding/UserLanding.jsx

```diff
@@ -46,15 +51,29 @@ const UserLanding = () => {
 
   const handleSubmit = async (e) => {
     e.preventDefault();
-    // const includesStrings = /workout|women/.test(userSearchTerm);
-    // const searchQuery = userSearchTerm + "workout" + "women";
-    // const searchResults = await getSearchVideos(searchQuery);
+
+    // const searchResults = await getSearchVideos(userSearchTerm);
+    
     // working with mock data
     const searchResults = searchResponse;
+    
+    setNextPageToken(searchResults.nextPageToken);
     setSearchedVideos(searchResults.items);
   };
 
-  const workoutVideos = savedWorkouts.map((obj) => (
+  const loadMoreSearch = async () => {
+    const searchResults = await getSearchVideos(userSearchTerm, nextPageToken);
+    setNextPageToken(searchResults.nextPageToken);
+    setSearchedVideos((prevVideos) => [...prevVideos, ...searchResults.items]);
+  };
+
+  const loadMoreWorkouts = () => {
+    setWorkoutsToShow((prevValue) => prevValue + 2);
```

Comment:
Love that we are limiting how many we show to the user at a time for both videos & workouts.

Now the next challenge if you want one is to add pagination on the BE to your load workouts query so we're truly "loading more" rather than loading it all at once and just hiding them until the user clicks :)